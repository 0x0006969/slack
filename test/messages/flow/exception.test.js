const Exception = require('../../../lib/messages/flow/exception');

describe('Exception', () => {
  test('works', async () => {
    const exception = new Exception('error-code-123', {
      team_id: 'T01234',
      user_id: 'U01234',
      channel_id: 'C01234',
    }).toJSON();
    expect(exception).toMatchSnapshot();
  });
});
