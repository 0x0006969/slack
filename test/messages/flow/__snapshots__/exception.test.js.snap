// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Exception renders generic error message 1`] = `
Object {
  "attachments": Array [
    Object {
      "color": "danger",
      "mrkdwn_in": Array [
        "text",
      ],
      "text": "Sorry, we had trouble with your request, but we're looking into it! If this is interfering with your work, please <https://github.com/contact?form%5Bsubject%5D=GitHub%2BSlack%20integration&form%5Bcomments%5D=Tell%20us%20a%20little%20more%20about%20what%20happened%20and%20include%20the%20error%20code%20below.%0A%0AError%20code%3A%20error-code-123|get in touch>!",
    },
  ],
  "response_type": "ephemeral",
}
`;
